### PR TITLE
Docs: Fix time range controls formatting

### LINKED ISSUE #19339: Fix Time Range Controls formatting
This topic has a few formatting issues: https://grafana.com/docs/reference/timerange/

* "Note:" should be bold.
* Ampersands should not be used as an abbreviation for "and".
* UI links should be bold, not in quotations.
* Any other grammar and clarity edits that you notice and I forgot to mention.

### PR DESCRIPTION
**Pull request by mondras**
**What this PR does / why we need it**: Tiume range controls documentation is out of date. Controls have moved and help was requested to update the docs (#19339)

**Which issue(s) this PR fixes**: #19339

Fixes #19339

**Special notes for your reviewer**:

Read the documentation regarding uploading images, since I don't have access to the grafana/website repository I'm attaching here the tinypng'd updated image for you to use it.

timepicker

### CODE REVIEW

File: docs/sources/reference/timerange.md [hunk 1]
```diff
@@ -70,4 +69,4 @@ When you zoom or change the Dashboard time to a custom absolute time range, all
 The `Hide time override info` option allows you to hide the override info text that is by default shown in the
 upper right of a panel when overridden time range options.
 
-Note: You can only override the dashboard time with relative time ranges. Absolute time ranges are not available.
+**Note**: You can only override the dashboard time with relative time ranges. Absolute time ranges are not available.
```
Comment (marcusolsson) [inline docs/sources/reference/timerange.md]:
```suggestion
**Note:** You can only override the dashboard time with relative time ranges. Absolute time ranges are not available.
```

Punctuation should have the same formatting as the text. Refer to [Formatting puncuation](https://docs.microsoft.com/en-us/style-guide/punctuation/formatting-punctuation).

**Review by marcusolsson — CHANGES_REQUESTED** [review]
Thanks! I'll have a look at updating the image as well!

**Comment by mondras** [comment]
Thanks for the suggestions on style. Already fixed them :)

**Review by marcusolsson — APPROVED** [review]

**Comment by marcusolsson** [comment]
LGTM!

**Pull request merged**